src/components/guidance-content/help/Glossary.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Grid, Row, ListGroup, ListGroupItem } from 'react-bootstrap';
+import { FullWidthCol, HalfWidthCol } from '../../layout/rb-derived-components';
+
+import _ from 'underscore';
+
+
+// Items are sorted automatically here.
+
```

Comment:
I made a list of possibly unfamiliar terms from text in the app. I don't know if we need these in the glossary, since they're all explained in the popovers.
- model
- emissions scenario
- variable
- dataset
- xlsx
- csv
- anomaly (if we keep using that name - one of my PRs renames it.)
- model run
- raster
- isoline
- layer

And two acronyms I didn't see any explanation for:
- GCM
- CMIP5

In summary, I'm not sureif we really need a glossary or what should go in it. Most things are already explained somewhere.

Reply:
Removed Glossary for now.